LFS171x/fabric-material/chaincode/tuna-app/tuna-chaincode.go

```diff
@@ -1,7 +1,10 @@
 // SPDX-License-Identifier: Apache-2.0
 
 /*
- * Sample Chaincode based on Demonstrated Scenario
+  Sample Chaincode based on Demonstrated Scenario
+
+ Parts of this code was contributed by the Hyperledger Team 
```

Comment:
Hyperledger Team --> Hyperledger Fabric community

Reply:
In addition, instead of contributed, we should use same phrasing that you used in the README. Something like "This code is based on code written by the Hyperledger Fabric community."

Reply:
done